src/components/shared/Slider/Slider.test.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { render, cleanup } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import { Slider } from './Slider';
+
+describe('Stars score component', () => {
+  afterEach(cleanup);
+
+  it('renders stars with score 0', () => {
```

Comment:
If I am not wrong, there is not any assert to verify that the score is `0`

Reply:
I have copied it from another test and I forgot to change the assert text, sorry.
Respect to the start value, the Slider could start with any value you want.